package.json

```diff
@@ -190,7 +190,7 @@
   "license": "GPL-3.0-or-later",
   "dependencies": {
     "@babel/runtime": "^7.17.0",
-    "@cowprotocol/contracts": "^1.1.3-RC.0",
+    "@cowprotocol/contracts": "^1.3.0",
```

Comment:
Curious about the versioning. From 1.1.3 to 1.3.0. Is this a typo or intentional?